src/client/services/conversation/share-activity/share-activity-base.js

```diff
@@ -118,9 +121,24 @@ var ShareActivityBase = SparkBase.extend(
    * @returns {Promise} Resolves with the updated activity
    */
   addFile: function addFile(file) {
+    /* eslint complexity: [0] */
```

Comment:
@ianwremmel I just wanted to acknowledge this.  You think this is because I was the last to add another path to the flow?